04_dataframe.ipynb

```diff
@@ -560,7 +560,7 @@
     "Dask.dataframe operations use `pandas` operations internally.  Generally they run at about the same speed except in the following two cases:\n",
     "\n",
     "1.  Dask introduces a bit of overhead, around 1ms per task.  This is usually negligible.\n",
-    "2.  When Pandas releases the GIL (coming to `groupby` in the next version) `dask.dataframe` can call several pandas operations in parallel within a process, increasing speed somewhat proportional to the number of cores. For operations which don't release the GIL, multiple processes would be needed to get the same speedup."
```

Comment:
I believe this was already done in https://github.com/pandas-dev/pandas/pull/10199.